src/pdn/README.md

```diff
@@ -411,8 +411,31 @@ add_pdn_connect -layers {metal9 metal10}
 pdngen
 ```
 
+### Useful developer functions
+
+If you are a developer, you might find these useful. More details can be found in the [source file](./src/PdnGen.cc) or the [swig file](./src/PdnGen.i).
+
+```tcl
+# source config file
+source_config config
+
+# get power nets
+get_pdn_power_nets
+
+# get ground nets
+get_pdn_ground_nets
+```
```

Comment:
these are leftover from the old PDNgen, so they are not really developer functions.
Possibly:
```
gui::debug_renderer 1/0 < enable disable
gui::debug_renderer_update
```

Reply:
Noted, just saw your PR on removing these deprecated functions. 

Currently, these GUI commands are missing, but I think they better fit in the GUI tool section. What do you think? @gadfort

Reply:
they are intended for developer use, so I think adding a GUI section to the developer section would be fine. General use users is discouraged.